### ISSUE TITLE
Preserve indentation in original output

### ISSUE THREAD

**Issue by alexbepple** [strongloop/node-foreman#62]
It looks as if _node-foreman_ trims everything. It should preserve indentation, as it might be meaningful.

I have a Procfile that does two things: 1) continuously compile LiveScript to JavaScript, 2) runs tests whenever a JavaScript file changes. The test output is indented, e.g. with Mocha’s spec reporter (http://mochajs.org/#reporters).

**Comment by jeffreywescott** [comment]
You might have a look at #64, which I just submitted to address this issue. You can use it in your package.json like so:

```json
  "devDependencies": {
    "foreman": "git+ssh://git@github.com/jeffreywescott/node-foreman.git#preserve-indentation"
}
```

**Comment by Darkhogg** [comment]
:+1: to this, I log a lot of objects for debugging and I kind of need this.

**Comment by rmg** [comment]
This was fixed in the 1.3.0 release a little while ago.
